src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/ResettlementAssessmentService.kt

```diff
@@ -40,4 +51,81 @@ class ResettlementAssessmentService(
       }
     }
   }
+
+  @Transactional
+  fun submitResettlementAssessmentByNomsId(nomsId: String, assessmentType: ResettlementAssessmentType, auth: String) {
+    // Check auth - must be NOMIS
+    val authSource = getClaimFromJWTToken(auth, "auth_source")?.lowercase()
+    if (authSource != "nomis") {
+      throw ServerWebInputException("Endpoint must be called with a user token with authSource of NOMIS")
+    }
+
+    val userId = getClaimFromJWTToken(auth, "user_id") ?: throw ResourceNotFoundException("user_id missing from user token")
```

Comment:
We need the user_id to be put in the message to SQS so we can generate the correct token to post to the Case Notes API. Get the username claim and use this in the database.